server_installation/topics/clustering/load-balancer.adoc

```diff
@@ -5,6 +5,11 @@ This section discusses a number of things you need to configure before you can p
 in front of your clustered {project_name} deployment.  It also covers configuring the built in load balancer that
 was <<_clustered-domain-example, Clustered Domain Example>>.
 
+The following diagram illustrates the use of a load balancer between clients and a {project_name} cluster. In another scenario, the load balancer could be a reverse proxy server.
+
+[[load-balancer-diagram]]
+.Example Load Balancer Diagram
+image:{project_images}/load_balancer.png[]
```

Comment:
Regarding the diagram, see my latest comment in JIRA: https://issues.redhat.com/browse/KEYCLOAK-10376?focusedCommentId=13937113&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-13937113